### PR TITLE
Add shortcut to toggle stats (FPS counter etc)

### PR DESCRIPTION
**Pull request by sigvef**
The shortcut is i.

### CODE REVIEW

File: nin/frontend/app/scripts/services/demo.js [hunk 1]
```diff
@@ -25,6 +26,16 @@ function demo(commands, $rootScope, $window) {
     stats[i].dom.style.float = 'right';
     stats[i].dom.style.display = 'block';
   }
+
+  commands.on('toggleStats', () => {
+    showStats = !showStats;
+    for(let i = 0; i < 3; i++) {
+      stats[i].dom.style.display = showStats ? 'block' : 'none';
```
Comment (aleksanb) [inline nin/frontend/app/scripts/services/demo.js]: If you use the current display style you can get away with no local style foro flipping the stats.

File: nin/frontend/app/scripts/services/demo.js [hunk 2]
```diff
@@ -25,6 +26,16 @@ function demo(commands, $rootScope, $window) {
...
+    }
+  });
+
+
+
```
Comment (aleksanb) [inline nin/frontend/app/scripts/services/demo.js]: Much space.

File: nin/frontend/app/scripts/services/demo.js (hunk 1 above)
Comment (sigvef) [inline nin/frontend/app/scripts/services/demo.js]: At the cost of making it less readable I guess. Updated.

File: nin/frontend/app/scripts/services/demo.js (hunk 2 above)
Comment (sigvef) [inline nin/frontend/app/scripts/services/demo.js]: Less space now.

**Comment by sigvef** [comment]
Updated.

**Pull request merged**